app.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+const path = require('path');
+
+const bodyParser = require('body-parser');
+const express = require('express');
+
+const config = require('./app/config/').Config;
```

Comment:
require('./app/config/') is looking for index file in app/config dir. And index.js in that dir exposing Config (see  app/config/index.js)